src/components/MultiHopTrade/components/TradeConfirm/TradeConfirmFooter.tsx

```diff
@@ -61,6 +63,26 @@ export const TradeConfirmFooter: FC<TradeConfirmFooterProps> = ({
     activeTradeId,
   })
 
+  const {
+    isLoading: isNetworkFeeCryptoBaseUnitLoading,
+    isRefetching: isNetworkFeeCryptoBaseUnitRefetching,
+    data: networkFeeCryptoBaseUnit,
+  } = useTradeNetworkFeeCryptoBaseUnit({
+    hopIndex: 0,
```

Comment:
@gomesalexandre does this break multi-hop? It seems like we'll now show the fee of the first hop for the second hop.

Reply:
@0xApotheosis what is multi-hop?